test/e2e/data/infrastructure-docker/main/clusterclass-quick-start.yaml

```diff
@@ -290,13 +302,11 @@ spec:
             controlPlane: true
         jsonPatches:
           - op: add
-            path: "/spec/template/spec/kubeadmConfigSpec/joinConfiguration/nodeRegistration/kubeletExtraArgs"
-            value:
-              cloud-provider: "external"
+            path: "/spec/template/spec/kubeadmConfigSpec/joinConfiguration/nodeRegistration/kubeletExtraArgs/cloud-provider"
+            value: "external"
```

Comment:
Are there things to keep in sync with our templates which get published? (also do it this way there so additional args could easily get added / reduce drifting away from the this template?)